rfcs/rfc-colin-states-mapping-to-lear.md

```diff
@@ -0,0 +1,154 @@
+            }
+            ...
+        },
+        Business.State.HISTORICAL: {
+            'restoration': { 
+                'fullRestoration': ['almagmation', 'continuationOut', 'continuationIn'], 
+                'limitedRestoration': ['almagmation', 'continuationOut', 'continuationIn'],
+            },
+            'putBackOn': ['almagmation', 'withdrawal']
+            ...
+        },
+        Business.State.LIQUIDATION: {
+            'restoration': {
+                'fullRestoration': ['liquidation'], 
+                'limitedRestoration': ['liquidation'], 
+            },
+            ...
+        }
+    },
+    'client': {
+        Business.State.ACTIVE: { # current business state
+            ...
+        },
+        Business.State.HISTORICAL: {
+            ...
+        },
+        Business.State.LIQUIDATION: {
+            'restoration': {
+                'fullRestoration': ['liquidation'], 
+                'limitedRestoration': ['liquidation'], 
+            },
+            'putBackOn': ['liquidation']
+            ...
+        }
+    }        
+    
+}
+```
+
+_Allowable state transitions matrix in COLIN for restoration and put back on filings_
+![colin_allowable_filings.png](rfc-colin-states-mapping-to-lear/colin_allowable_filings.png)
+
+Filing validation logic will be updated to include logic to ensure that only valid business state transitions are allowed. 
+
+All filings resulting in business state transitions will need to populate the businesses.state_filing_id field.  This is used to determine if future business state transitions are permitted.
+
+Filings.meta_data field for any filings where a business state transitions occurs will include from/to state.
+
+API consumers such as the FE with a need to determine which filing transitioned the business to its current state will retrieve the business state filing provided by the business response of the GET business endpoint.  For example, this can be used to determine if an ACTIVE business is currently in _limited restoration_.
+
+# Adoption strategy
+
+The _LEAR_ team will create tickets to implement the required components.
+
+# Unresolved questions
+
+There are still some details that need to be fleshed out for filings like the ones below. These details will be worked out as we implement the filings in LEAR.
+
+**Involuntary dissolution** - this is currently a manual process that transitions a business to D1 to D2 based on some conditions. Uncertain as to how this changes in LEAR and how we will represent the involuntary dissolution process from a data and process perspective. Will the manual process become an automated process in LEAR?
+
+**Liquidation** - how does this process look? do we need additional business states for the liquidation steps or will we be able to just use the business state of “LIQUIDATION” and determine which step in the liquidation process a business is in via some dynamic calculations?
```

Comment:
I know I listed it as a possible state, but it isn't a state. It's another a flow and flow_statuses that like _Good Standing_ transitions a business to a **historical** state.

Reply:
Added clarification in basic design section that "liquidation" is not a business state.  Also removed liquidation question from this section of the RFC.